### ISSUE TITLE
[Accessibillity] Workbench: Button does not have an accessible name

### ISSUE THREAD

**Issue by nwanduka** [camicroscope/caMicroscope#982]
**Describe the bug**
The foward button doesn't have an accessible name. Screen readers do not recognize it on hover and on click it reads as just "button"

**Expected behavior**
The button should have an accessible name that describes it's function. In this case "Select or create your dataset"

**Screenshots**
![image](https://github.com/camicroscope/caMicroscope/assets/122698422/bc4cec3b-1f6b-4f5b-80e2-61949392c7ad)
